src/v2/view-builder/views/password/ReEnrollAuthenticatorPasswordView.js

```diff
@@ -14,7 +14,7 @@ const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
   },
 
   getPasswordPolicy () { 
-    return this.options.appState.get('recoveryFactor').settings;
+    return this.options.appState.get('recoveryAuthenticator').settings;
```

Comment:
Do we need to use it something like this ?
`return this.options.appState.get('recoveryAuthenticator').settings || this.options.appState.get('recoveryFactor').settings;`

Because if `AUTHENTICATOR_PLATFORM` is ON, it will be `recoveryAuthenticator` and if it is `OFF`, it will be `recoveryFactor`? Or should we just ignore factors?

Reference: https://okta.slack.com/archives/CCA77QVL2/p1593714897203000

Reply:
when `AUTHENTICATOR_PLATFORM` is OFF, it shall hint `ReEnrollFactorPasswordView` which doesn't need exists.